src/softphone.js

```diff
@@ -47,7 +47,7 @@
         },
         failure: function (reason) {
           if (reason.message && reason.message.includes("SoftphoneConnectionLimitBreachedException")) {
-            publishError("multiple_softphone_active_sessions", "Number of active sessions are more then allowed limit.");
+            publishError("multiple_softphone_active_sessions", "Number of active sessions are more then allowed limit.", "");
```

Comment:
Found out all other usages of `publishError()` used three arguments. In order to guarantee the `TypeScript` typings (i.e. non-null/undefined), updated this call.